libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -432,7 +428,8 @@ def __ListByLabel(
       raise RuntimeError(error_msg)
 
     # pylint: disable=line-too-long
-    resource_dict = {}  # type: Dict[str, Union[GoogleComputeInstance, GoogleComputeDisk]]
+    resource_dict = {
+    }  # type: Dict[str, Union[GoogleComputeInstance, GoogleComputeDisk]]
```

Comment:
If line-too-long no need to split the dict :D

Reply:
that is definitely a remnant of  my "out of mistake" yapf check that I undid manually

Reply:
Fixed it